alibi_detect/cd/tensorflow/classifier.py

```diff
@@ -51,8 +52,7 @@ def __init__(
         x_ref
             Data used as reference distribution.
         model
-            TensorFlow classification model used for drift detection. If this is a subclassed model, it should be
-            built/called before passing it to the detector.
+            TensorFlow classification model used for drift detection.
```

Comment:
I don't fully follow, can you clone if it's not built? I thought that was the original issue why the model needed to be built.

Reply:
There are two main limitations with subclassed models:

1. Cloning is a problem if the correct `custom_objects` were not passed during loading. See discussion in https://github.com/SeldonIO/alibi-detect/pull/628.
2. `get_weights()` will return a length zero list which causes `set_weights()` to fail, if the model has not been built. 

Removing the cloning in this PR and replacing with getting/setting weights would resolve the problem in https://github.com/SeldonIO/alibi-detect/pull/628, in that users would be able to instantiate and run detectors without worrying about providing `custom_objects` at load time. However, due to 2. above, it would add a different limitation, in that models would have to be built before passing to the detector, so that the getting/setting of weights would work. On balance, I think the latter (having to build the model) is more of a limitation than the former (providing custom objects). Although this is subjective...

Reply:
P.s. the above change changes the docstring back to what it is in `master`